src/coreclr/src/vm/pefile.cpp

```diff
@@ -2475,3 +2474,116 @@ PTR_ICLRPrivBinder PEFile::GetBindingContext()
 
     return pBindingContext;
 }
+
+// Function to get the fully qualified name of an assembly
+void PEAssembly::GetFullyQualifiedAssemblyName(IMDInternalImport* pImport, mdAssembly mda, SString& result, DWORD flags)
+{
+    CONTRACTL
+    {
+        PRECONDITION(CheckValue(result));
+#ifndef DACCESS_COMPILE
+        THROWS;
+#else
+        NOTHROW;
+ #endif // !DACCESS_COMPILE
+        GC_TRIGGERS;
+        MODE_ANY;
+    }
+    CONTRACTL_END;
+
+    if (pImport != NULL)
+    {
+        // This is for DAC, ONLY for the binding tool.  Don't use for other
```

Comment:
Is this comment still correct, since we don't call into fusion any more

Reply:
Yeah, the binding tool was a prototype that never shipped. Deleted the whole method.